app/api/lesswrongAuth/route.ts

```diff
@@ -0,0 +1,180 @@
+import { NextRequest } from "next/server";
+import { cookies } from "next/headers";
+import { createClient } from "@/utils/supabase/server";
+import { GetUser } from "@/utils/userData";
+
+const lesswrongEndpoint = "https://www.lesswrong.com/graphql";
+
+interface lesswrongApiResponse<T> {
```

Comment:
(i'll allow it for code velocity) style throughout codebase has been interfaces in `CamelCase` rather than `camelCase`